core/src/test/java/hudson/util/XStream2Test.java

```diff
@@ -296,11 +296,6 @@ public void concurrentHashMapSerialization() throws Exception {
         } finally {
             v.delete();
         }
-
-        // should be able to read in old data just fine
```

Comment:
The Javadoc for the test is now incorrect—it is no longer testing that we

> still can deserialize to [from?] older, verbose format

Since it is no longer testing Jenkins code at all, I would rather suggest just deleting the whole test case—the part you left behind was only a kind of control case for the real test which is now deleted.